var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -25,9 +25,14 @@ class Neuron(Package):
 
     # Patch which reverts 81a7a39 for numerical compat
     patch('revert_Import3d_numerical_format.patch', when='@7.8.0:')
+    # Patch which reverts d9605cb for not hanging on ExperimentalMechComplex
+    patch('revert_d9605cb.patch', when='@7.8.0a')
+
+    # Patch which applies load balancing fix
+    patch('apply_79a4d2af_load_balance_fix.patch', when='@7.8.0:')
```

Comment:
The patches look nearly identical to me. Any chance we could re-use the same patch? Or at least name them similarly, so that it is clear that they are for different versions?

Reply:
Patches are automatically generated, and serve different purposes. And Im too afraid of doing any manual change or marge